### ISSUE TITLE
the acc problem

### ISSUE THREAD

**Issue by czqhubu** [AIrocker/MobileFaceNet_Tutorial_Pytorch#2]
thank you for your share, but I trained the code ,and after epoch 1, the loss and the acc is not same to you log data , can you share you train params , such batch and so on
you logs:Epoch 0/11, Iters: 045400, loss: 10.7383, train_accuracy: 0.0391, time: 0.38 s/iter, learning rate: 0.01
my logs:Epoch 0/11, Iters: 003000, loss: 23.2479, train_accuracy: 0.0000, time: 0.07 s/iter, learning rate: 0.01
